usaspending_api/search/v2/urls_search.py

```diff
@@ -17,4 +17,5 @@
     url(r'^spending_by_transaction_count', SpendingByTransactionCountVisualizaitonViewSet.as_view()),
     url(r'^spending_by_transaction', SpendingByTransactionVisualizationViewSet.as_view()),
     url(r'^transaction_spending_summary', TransactionSummaryVisualizationViewSet.as_view()),
+    url(r'^transaction_sum', SpendingByTransactionSumVisualizaitonViewSet.as_view())
```

Comment:
Please remove all this extraneous code pulled from an earlier commit of the `transaction_sum` PR